libsolidity/ast/Types.cpp

```diff
@@ -2874,53 +2874,59 @@ TypePointers FunctionType::parameterTypes() const
 	return TypePointers(m_parameterTypes.cbegin() + 1, m_parameterTypes.cend());
 }
 
+namespace {
+string to_string(FunctionType::Kind _kind)
```

Comment:
Could also follow the pattern of `stateMutabilityToString` and have `functionTypeKindToString`.

Reply:
@chriseth so you prefer the `to_string` as opposed to the other suggestions? (I do, but we use both interchangeably)

Reply:
I don't care much.